test/neg-http.mjs

```diff
@@ -556,13 +574,19 @@ describe('Testing the challenge http API', () => {
     // if the queue is still running, let's close down after it ends
     // if it's empty, close down immediately
     if (txQueue.length === 0) {
+      console.log('neg-http.mjs after - test suits - if case');
```

Comment:
can we remove some of the debug `console.log` in tests. It will de-clutter the output

Reply:
removed